### DISCUSSION TITLE
Checking my understanding re: BERTopic 'topic_model.transform()'

### DISCUSSION THREAD

**Discussion by registjl** [MaartenGr/BERTopic#2169] [General]
Hi -- I'm a BERTopic newbie so please bear with me.  

I want to confirm my understanding of the results from using 'topic_model.transform()' function.
**Quick overview of my process:** 
1. I've instantiated a BERTopic model using a pre-trained embedding model 
`topic_model = BERTopic(umap_model=umap_model, hdbscan_model=hdbscan_model,
                       calculate_probabilities=True, embedding_model=embedding_model)`

2. I fit my topic_model with 'fit_transform' with my training data and save the model
`topics, probabilities = topic_model.fit_transform(TRAIN_DATA[text])
topic_model.save(model_name)`

3. In another script I load the saved topic_model and use 'topic_model.transform' with new data.
`topic_model = BERTopic.load(topic_model_name)
topics, probabilities = topic_model.transform(TEST_DATA)`

**MY QUESTION:  The topics returned in step 3 are the "predicted" topics that each document in the TEST_DATA is expected to be part of.   Is that right?**  

- In other words, there aren't any "new topics" created -- just assignment to one in the original model?

**Comment by MaartenGr** [comment]
> MY QUESTION: The topics returned in step 3 are the "predicted" topics that each document in the TEST_DATA is expected to be part of. Is that right?

That is correct!

> In other words, there aren't any "new topics" created -- just assignment to one in the original model?

Right again, no new topics are created with `.transform`.
